Task_DEV_4/Task_DEV_4/Figura.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Task_DEV_4
+{
+    public abstract class Figure
```

Comment:
Class name was changed but in the name of file typo remains.

Reply:
Done.